modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -4,15 +4,9 @@
 
 import * as runtime from '../runtime';
 {{#imports.0}}
-import {
-    {{#imports}}
-    {{className}},
-    {{^withoutRuntimeChecks}}
-    {{className}}FromJSON,
-    {{className}}ToJSON,
-    {{/withoutRuntimeChecks}}
-    {{/imports}}
-} from '../models';
+// Some imports not used depending on template conditions
```

Comment:
**suggestion:** "template condition" is something specific to openapi-generator.
However, this comment is gonna end up in the generated code, a part of end user's app - where it doesn't really make much sense.

Can we come up with a more generic copy (e.g. `Some imports might not be used`)?

Reply:
Yeah, sure. I'll change it.